cmd/tailscaled/depaware.txt

```diff
@@ -71,6 +71,7 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
         github.com/golang/groupcache/lru                             from tailscale.com/net/dnscache
         github.com/google/btree                                      from gvisor.dev/gvisor/pkg/tcpip/header+
         github.com/hdevalence/ed25519consensus                       from tailscale.com/tka
+   L 💣 github.com/illarion/gonotify                                 from tailscale.com/net/dns
```

Comment:
I assume you looked at the use of unsafe and weren't horrified?

Reply:
Yeah, it's exactly what I expected. Nice minimal wrapper package. One little unsafe.Pointer use to copy things out of the buffer, like we have in util/dirwalk too.